static/js/desktopStatistics.js

```diff
@@ -81,71 +94,93 @@ function manipulateData(data, options) {
 
 function colorShade(usageRange, colors) {
   var index = 0;
-  if (usageRange <= 1) {
+  if (usageRange <= 0.003326016523) {
```

Comment:
I'm not sure this is right. I changed the take before but was told to use the snapcraft range

Reply:
I bet snapcraft has it wrong, I broke it into 5 percentiles, which is a truer visualisation.  I looked at lots of info on this, but I will chat with Juan